<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Pirogov, Sokolova, Bubencov, Bogomolov</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Вариант 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>P(Вероятность)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V(Скорость перебора)</string>
    </property>
   </widget>
   <widget class="QLabel" name="PowerAlphabet">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A(Мощность алфавита)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>T(Срок действия пароля)</string>
    </property>
   </widget>
   <widget class="QLabel" name="LowerCasePassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>S*(Нижняя граница паролей) </string>
    </property>
   </widget>
   <widget class="QLabel" name="LengthPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>L(Длина пароля)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chanceLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>0.000001</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="speedLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="termLine">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="CreatedPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерированный пароль</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>111</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="LowerCasePassword_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>220</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Alphabet">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LengthPassword_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CreatedPassword_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
